src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -956,12 +954,7 @@ export function submitAppointmentOrRequest(history) {
           ...additionalEventData,
         });
         resetDataLayer();
-
-        if (featureBreadcrumbUrlUpdate) {
-          history.push(`/${appointment.id}?confirmMsg=true`);
-        } else {
-          history.push(`/va/${appointment.id}?confirmMsg=true`);
```

Comment:
Now that we have removed the last remaining redirect to /va/:id, let's remove it in the route configurations [here](https://github.com/department-of-veterans-affairs/vets-website/blob/0780ba4edd42d616b411a604db16d4ba8adbfb9e/src/applications/vaos/appointment-list/index.jsx#L41C20-L41C23).